.github/workflows/deploy.yml

```diff
@@ -45,28 +48,64 @@ jobs:
           sushi --version
           sushi .
 
+
+    # Install GOFSH if needed
+      - name: "Install GOFSH"
+        shell: bash
+        run: |
+          npm install -g gofsh@1.6.4
+          gofsh --version
+
+    # Install reproschema2fhir if needed
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: kind-lab/reproschema-to-fhir
+          submodules: 'true'
+          path: './tofsh'
+          ssh-key: ${{ secrets.SSH_DEPLOY_KEY }}
+    
+    # Call Gofsh if needed
+      - name: Checkout
+        shell: bash
+        run: |
+          cd ./tofsh
+          pip install -e .
+          cp .env.example .env
+          git submodule update --init --recursive
+          echo -e "CODESYSTEM_URI = https://voicecollab.ai/fhir/CodeSystem/\nVALUESET_URI = https://voicecollab.ai/fhir/ValueSet/\nQUESTIONNAIRE_URI = https://voicecollab.ai/fhir/Questionnaire/\nQUESTIONNAIRE_LANGUAGE = en\nFHIR_QUESTIONNAIRE_MODE = AnswerOptions" >> .env
```

Comment:
i think it's more readable to split this up into multiple echo calls, rather than have them combined together with `\n`